tests/jerry/regression-test-issue-3325.js

```diff
@@ -0,0 +1,19 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var text = 'x';
+var index = Math.random () - 1;
+assert (text.charAt(0) === "x");
+assert (text.charAt(index) === "x");
+assert (text.charAt(index) !== "");
```

Comment:
We shouldn't use Math.random() in this test case for two reasons:
- This test can fail intermittently, because index is -1 if Math.random() is 0.
- This bug doesn't depend on random things at all, it's more than enough
testing -0.1, -0.5 and -0.9 indexes.